### PR TITLE
[MST] Update security page with content from Kbase

### PR DESCRIPTION
**Pull request by Loquacity**
# Description

Added extra content from Kbase, but also did a thorough edit of this, it was pretty old and crusty.

~@alexthinath please see my question in Slack re: the Aiven cybersecurity report, before approving this PR.~

# Links

Fixes #[insert issue link, if any]

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/timescaledb/latest/contribute-to-docs/)

# Review checklists
Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

- [ ] Is the content technically accurate?
- [ ] Is the content complete?
- [ ] Is the content presented in a logical order?
- [ ] Does the content use appropriate names for features and products?
- [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

- [ ] Is the content free from typos?
- [ ] Does the content use plain English?
- [ ] Does the content contain clear sections for concepts, tasks, and references?
- [ ] Have any images been uploaded to the correct location, and are resolvable?
- [ ] If the page index was updated, are redirects required
      and have they been implemented?
- [ ] Have you checked the built version of this content?

### CODE REVIEW

**Comment by alexthinath** [comment]
Copying from Slack:

```
Given that we provide that link as "read our latest evaluation report", and it's from 2017, makes me a bit :grimacing:
[1:21 AM] do we have something more recent? or, even better, a link that will be updated regularly?
```

I'll ask Aiven, this came up recently. I've asked Aiven. That linked report is current as of 2019 though

**Review by charislam — APPROVED** [review]

**Comment by Loquacity** [comment]
> Copying from Slack:
> 
> ```
> Given that we provide that link as "read our latest evaluation report", and it's from 2017, makes me a bit :grimacing:
> [1:21 AM] do we have something more recent? or, even better, a link that will be updated regularly?
> ```
> 
> I'll ask Aiven, this came up recently. I've asked Aiven. That linked report is current as of 2019 though

That's unfortunate 😞

File: mst/security.md [hunk 1]
```diff
@@ -5,162 +5,156 @@ product: mst
...
+firewall based on iptables, which only allows connections from specific
+addresses. This is used for network traffic from the internal network to other
+VMs in the same service, and for external public network, to client connections.
+
+By default, new services accept incooming traffic from all sources, which is
+used to simplify initial set up of your service. It is highly recommended that
+you restrict the IP addresses that are allowed to establish connections to your
+services.
+
+<prcoedure>
```
Comment (Loquacity) [inline mst/security.md:97]:
```suggestion
<procedure>
```

File: mst/security.md [hunk 2]
```diff
@@ -5,162 +5,162 @@ product: mst
...
-turn encrypted with RSA key-encryption key-pair and stored in the header section
-of each backup segment. The file encryption is performed with AES-256 in CTR
-mode with HMAC-SHA256 for integrity protection. The RSA key-pair is randomly
-generated for each service. The key lengths are 256-bit for block encryption,
-512-bit for the integrity protection and 3072-bits for the RSA key.
+turn encrypted with an RSA key-encryption key-pair, and stored in the header
+section of each backup segment. The file encryption is performed with AES-256 in
+CTR mode, with HMAC-SHA256 for integrity protection. The RSA key-pair is
+randomly generated for each service. The key lengths are 256-bit for block
+encryption, 512-bit for the integrity protection and 3072-bits for the RSA key.
```
Comment (charislam) [inline mst/security.md:78]:
```suggestion
encryption, 512-bit for the integrity protection, and 3072-bits for the RSA key.
```

**Review by charislam — APPROVED** [review]

**Pull request merged**